starter/src/main/java/io/javaoperatorsdk/operator/springboot/starter/CrdUploader.java

```diff
@@ -0,0 +1,91 @@
+package io.javaoperatorsdk.operator.springboot.starter;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.function.UnaryOperator;
+
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.fabric8.kubernetes.api.model.ObjectMeta;
+import io.fabric8.kubernetes.client.KubernetesClient;
+
+import static java.util.Arrays.stream;
+import static java.util.Objects.requireNonNull;
+import static java.util.stream.Collectors.joining;
+
+public class CrdUploader {
```

Comment:
Little nitpicking about naming, this is not really az upload, maybe `CRDApplier`?